src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -120,7 +121,7 @@
    (fn [{:keys [state props refs]}]
      [:div {}
       (when (:show-import-mc-modal? @state)
-        (render-import-modal state props refs))
+        (render-import-modal state props refs (:onImportFunc props)))
```

Comment:
Just leave this in props, no need to pull it out and pass it around as an extra parameter.